styx-standalone-service/src/main/resources/styx-standalone.conf

```diff
@@ -18,6 +18,7 @@ styx.stale-state-ttls = {
 styx.gke.default.project-id = ""
 styx.gke.default.cluster-zone = ""
 styx.gke.default.cluster-id = ""
+styx.gke.default.namespace = "default"
```

Comment:
Add to internal conf afterwards?

Reply:
sure, there is a default fallback in the code though

Reply:
Better to be more explicit. :)